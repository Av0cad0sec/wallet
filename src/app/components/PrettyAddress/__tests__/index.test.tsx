import * as React from 'react'
import { render } from '@testing-library/react'

import { PrettyAddress } from '..'

describe('<PrettyAddress  />', () => {
  it('should match snapshot', () => {
    const page = render(<PrettyAddress address="oasis1aaaaaaaaaaaa" />)
    expect(page.container.firstChild).toMatchSnapshot()
  })

  it('should split the address', () => {
    const page = render(<PrettyAddress address="oasis1aaaaaaaaaaaa" />)
    expect(page.container.firstChild).toMatchSnapshot()
    expect(page.container.firstChild!.textContent).toMatch(/oasis1 aaaa aaaa aaaa/)
  })
})
