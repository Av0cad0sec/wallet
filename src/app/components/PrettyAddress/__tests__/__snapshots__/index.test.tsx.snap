// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PrettyAddress  /> should match snapshot 1`] = `oasis1 aaaa aaaa aaaa`;

exports[`<PrettyAddress  /> should split the address 1`] = `oasis1 aaaa aaaa aaaa`;
